test-requirements.txt

```diff
@@ -1,4 +1,10 @@
 # These requirements must be installable on all our supported versions
+
+# This is the last version of httpx compatible with Python 3.6
```

Comment:
do we need to add these to setup.py?

Reply:
I don't think so. This is like `requests` -- we don't *require* it but we use it if the user happens to have it installed.